src/main.iced

```diff
@@ -41,6 +42,15 @@ output = ({outfile, code}, cb) ->
 
 #================================================
 
+merge_asts = (ast1, ast2) ->
+  _.mergeWith ast1, ast2, (value1, value2) ->
+    if Array.isArray(value1)
+      return value1.concat value2
+    else
+      return undefined
```

Comment:
`return undefined` is actually required by Lodash [mergeWith](https://lodash.com/docs/4.17.15#mergeWith). It means that lodash should fallback to its default merge algorithm. Lodash merge doesn't concatenate arrays by default in a merge, so that's the only custom behavior we want.